### ISSUE TITLE
Remove Gen 2 devices

### ISSUE THREAD

**Issue by Subivan1** [nschum/homebridge-twinkly#28]
Is it possible to remove /exclude gen 2 devices? I have just upgraded all my lights to latest firmware and the gen 2 lights are now HomeKit compatible. This causes duplicate devices within HomeKit. My gen 1 devices still need to work with this plugin.
Are there any options or features that could be added to specify which twinkly devices should be included or excluded in the plugin?

**Comment by JacobSyndeo** [comment]
+1; this will be an important feature for this plugin moving forward.

**Comment by inspectorgadjet7** [comment]
Agreed. But it turns out that not all Twinky Gen II devices are supported. If your device is a ‘Polygroup’ licensed product like my pre-lit tree, HomeKit is not currently available, so this feature would need to either detect a Polygroup product somehow, or just let us hide certain devices.

**Comment by JacobSyndeo** [comment]
Let’s just be able to hide certain devices then.

**Comment by vanHoesel** [comment]
I have made some tweaks to this plugin that only now show up in the official HomeKit firmware upgrade ... but I have more to come.

Dropping Gen-II is not a wise idea, but maybe a option to turn it off

**Comment by nsforge** [comment]
I just bought some Gen 2 lights, but the built-in HomeKit support seems very broken. When I enabled it, the lights became glitchy and juddery, and the responsiveness to commands through the app dropped to 1-2s (and occasionally just timed out completely). Power cycling didn't help, and the only way I was able to fix it was to do a factory reset.

Given that the 1st-party support seems to be somewhat broken, I'd like to try using homebridge instead (since that shouldn't have any impact on the devices's onboard CPU/RAM resources), so I'd like to add another vote for adding the ability to hide devices, rather than actually dropping support for Gen 2 devices.

**Comment by vanHoesel** [comment]
By design, it is now doing a auto-detect, which is awesome, Apple Homekit should all be easy to use.

I still rely on manual configuration and have my strings even hardcoded in my router to assign an IP-address. I have listed my strings in the config file.

For 'heavy-users' I would advice to add a 'blacklist' option for IP addresses
